frappe/core/doctype/communication/communication.json

```diff
@@ -401,7 +401,8 @@
  "icon": "fa fa-comment",
  "idx": 1,
  "links": [],
- "modified": "2023-11-27 20:38:27.467076",
+ "make_attachments_public": 1,
```

Comment:
This is not ideal :grimacing:

Reply:
Need some time to figure out inline attachments just like pdfs.

Reply:
Yup struck me later. I was initially trying to keep the original behaviour.